styles.css

```diff
@@ -60,6 +60,68 @@ img {
 
 /*=== main section ===*/
 
+.idea-cards {
```

Comment:
Nice grid work! Really glad to see this in action as I have little experience with grid.